scripts/world_bank/health_nutrition_population/HealthcareExpenditure_README.md

```diff
@@ -0,0 +1,83 @@
+Importing World Bank Healthcare Expenditure Data into Data Commons
```

Comment:
Since this is currently population only, we can rename the file to just 'README.md' and change the title to Importing World Bank Population Data into Data Commons.

When we have more data, we can revise the title to include other data sets.